Makefile

```diff
@@ -7,5 +7,35 @@ pylint:
 	pylint nameko_sqlalchemy -E
 
 pytest:
-	coverage run --concurrency=eventlet --source nameko_sqlalchemy --branch -m pytest test
+	coverage run --concurrency=eventlet --source nameko_sqlalchemy --branch -m \
+		pytest test \
+			--test-db-url="mysql+pymysql://test_user:password@$(shell docker port nameko_sqlalchemy_test_toxiproxy 3307)/nameko_sqlalchemy_test" \
+			--toxiproxy-api-url=$(shell docker port nameko_sqlalchemy_test_toxiproxy 8474)
 	coverage report --show-missing --fail-under=100
+
+setup-containers:
+	-make cleanup
+	make start-containers
+	make setup-toxiproxy
+	make wait-for-mysql
```

Comment:
You can structure the `Makefile` to specify the dependencies inline:

``` Makefile
pytest: test-deps
    coverage run ...

test-deps: container-cleanup mysql-setup toxiproxy-setup

mysql-setup: mysql-container
    @echo Waiting for mysql to start
    ...

toxiproxy-setup: toxiproxy-container
    @echo Setting up toxiproxy to mysql
    ...

mysql-container:
    docker run...

toxiproxy-container:
    docker run...

container-cleanup:
    @echo Removing docker containers...
    ...
```

(targets to the right of the colon are dependencies of the target to the left)